engine/apps/mattermost/tests/test_utils.py

```diff
@@ -0,0 +1,26 @@
+import pytest
+from django.conf import settings
+
+from apps.mattermost.exceptions import MattermostEventTokenInvalid
+from apps.mattermost.utils import MattermostEventAuthenticator
+
+
+@pytest.mark.django_db
+def test_jwt_token_validation_success(
+    make_organization,
+):
+    organization = make_organization()
+    token = MattermostEventAuthenticator.create_token(organization=organization)
+    payload = MattermostEventAuthenticator.verify(token)
+    assert payload["organization_id"] == organization.public_primary_key
+
+
+@pytest.mark.django_db
+def test_jwt_token_validation_failure(
+    make_organization,
```

Comment:
You should probably use the pytest fixture [settings](https://pytest-django.readthedocs.io/en/latest/configuring_django.html#overriding-individual-settings) for the override here.